### PR TITLE
Converts Captain's space suit into a hardsuit

### PR DESCRIPTION
**Pull request by Spess-Rando**
:cl:
tweak: Captain's space armor has been readapted with modern hardsuit technology.
/:cl:

Turns the existing captain's space suit into captain's hardsuit.

Aside from changes that are inherent with being a hardsuit, it's the same as ever - its stats remain unchanged.

Also, now drones can no longer steal the helmet, which is probably a good thing.

### CODE REVIEW

**Comment by LanCartwright** [comment]
captain is a veteran
captain requires old school retro gear that they just don't make them like they used to
thus antique laser gun, spacesuit

**Comment by Sligneris** [comment]
Nanotrasen is aware of the Captain's tastes, which is why the space armor's design was not changed.

It is not at all caused by budget cuts in hardsuit design department, or by the lack of spriters. No, sire! We are trying hard to keep your retro experience intact.

**Comment by nullbear** [comment]
instead of a sechailer (which promotes cap acting as shitcurity) give him an alternate version which shouts 'productivity motivators' to (who are we kidding here) promote better captain behaviour.

**Comment by Super3222** [comment]
From a player perspective, I always looked at the Captains hardsuit (compared to every other hardsuit) as a tribute towards when helmets and hardsuits weren't connected.

But on the other hand, less drones fucking with your shit, so I'm sold.
Why change the gas mask in there to a sechailer? Seems out of the Captain's style.

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -32,9 +32,8 @@
 	mask_type = /obj/item/clothing/mask/breath
 
 /obj/machinery/suit_storage_unit/captain
-	suit_type = /obj/item/clothing/suit/space/captain
-	helmet_type = /obj/item/clothing/head/helmet/space/captain
-	mask_type = /obj/item/clothing/mask/gas
+	suit_type = /obj/item/clothing/suit/space/hardsuit/captain
+	mask_type = /obj/item/clothing/mask/gas/sechailer
```
Comment (KorPhaeron) [inline code/game/machinery/suit_storage_unit.dm]: the sprite is meant to go with a gas mask

**Comment by InsaneHyena** [comment]
I don't know why nobody did it sooner.

**Comment by AnturK** [comment]
Without actually changing sprite this is kinda eh.

File: code/game/machinery/suit_storage_unit.dm (hunk 1 above)
Comment (Sligneris) [inline code/game/machinery/suit_storage_unit.dm]: The sprite actually goes well with either. 

That part's mostly an aesthetic detail, I guess. Think I should drop that out?

**Comment by Niknakflak** [comment]
Yea. Eh

**Comment by Sligneris** [comment]
I mean, anyone is free to update the sprite later on.

**Comment by Jalleo** [comment]
This is done as a buff against Tesla.

**Comment by GunHog** [comment]
At the very least make a new helmet sprite which supports lights.

**Comment by lzimann** [comment]
@jalleo after explosive tesla hardsuits wont save you from hell

**Comment by Bawhoppen** [comment]
I liked being able to wear the helmet on its own

**Comment by Dorsisdwarf** [comment]
I liked being able to steal it as an assistant and run around yelling HAT HAT HAT at security

:-1:

**Comment by MrStonedOne** [comment]
little differences like the helmet being removable are cool, and this need to make everything fit into established sanitary molds is rather lame and monogamous.

**Comment by optimumtact** [comment]
I just hate hardsuits

**Pull request merged**